joss-paper/paper.md

```diff
@@ -0,0 +1,123 @@
+---
+title: '21cmFAST v3: A fully Python-integrated simulation code to generate 3D 21cm fluctuation fields through cosmic history.'
+tags:
+  - Python
+  - astronomy
+  - cosmology
+  - simulation
+authors:
+  - name: Steven G. Murray
+    orcid: 0000-0003-3059-3823
+    affiliation: 1
+  - name: Bradley Greig
+    orcid: xxxxxx
+    affiliation: 2
+  - name: Andrei Mesinger
+    orcid: xxxxxx
+    affiliation: 3
+affiliations:
+ - name: School of Earth and Space Exploration, Arizona State University, Phoenix, USA
+   index: 1
+date: 29 Feb 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+The field of 21cm cosmology -- in which the hyperfine spectral line of neutral hydrogen
+is mapped over large swathes of the Universe's history -- has developed radically over
+the last decade.
+Notwithstanding the overwhelming technical challenges associated with the observation
+and eventual detection of this signal, the promise of the field is to revolutionise our
+knowledge of the first light sources: a worthy prize indeed.
```

Comment:
a worthy prize indeed -> through the timing and patterns they imprint in the cosmic 21-cm signal

first light sources -> first stars, galaxies, and black holes

Reply:
Cool, did that.